Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.9.4
```

Comment:
1.10? You could also consider using a multi-stage build to reduce the final image size.